### PR TITLE
Updating Useful Terms

### PR DESCRIPTION
**Pull request by abelinkinbio**
Clarifies the definition of a Tunnel + Connector. Adds a description for Replicas.

### CODE REVIEW

File: content/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms.md [hunk 1]
```diff
@@ -24,7 +24,11 @@ The `cloudflared tunnel create <NAME>` command creates a tunnel and assigns it a
...
+  
+## Replica
+
+You can create and configure a tunnel once and run that tunnel through multiple, unique instances of the connector, `cloudflared`. These instances are known as connectors, or replicas. DNS records and Cloudflare Load Balancers will still point to the tunnel and its DNS Record (UUID.cfargotunnel.com), while that tunnel sends traffic to the multiple instances of cloudflared that run through it. Today, there is no guarentee about which connector will be chosen. Replicas are often deployed to provide tunnels with high availability in the event a given host running cloudflared is interupted or taken offline.
```
Comment (DevinCarr) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms.md]: s/guarentee/guarantee

Comment (abelinkinbio) [reply content/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms.md]: Good catch 😅

**Review by DevinCarr — APPROVED** [review]

**Review by nmldiegues — APPROVED** [review]

File: content/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms.md [hunk 2]
```diff
@@ -24,7 +24,11 @@ The `cloudflared tunnel create <NAME>` command creates a tunnel and assigns it a
 
 ## Connector
 
-You can create and configure a tunnel once and run it as multiple different `cloudflared` processes. These processes are known as connectors, or replicas. DNS records and Cloudflare Load Balancers can still point to the tunnel and its UUID, while that tunnel sends traffic to the multiple instances of cloudflared that run through it. Using multiple connectors provides tunnels with high availability, scalability, and elasticity.
+The connector, simply called `cloudflared`, establishes connectivity from your origin server to the Cloudflare global network. Each instance of the connector will be given a unique connector id which can be surfaced by running `cloudflared tunnel info <NAME or UUID>` or visiting the Zero Trust dashboard. Our connector offers high-availability by design, creating four long-lived connections to two distinct data centers within Cloudflare’s global network. This means that whether an individual connection, server, or data center goes down, your origin remains available.
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms.md]:
```suggestion
The connector, referred to as `cloudflared`, establishes connectivity from your origin server to the Cloudflare global network. Our connector offers high availability by creating four long-lived connections to two distinct data centers within Cloudflare’s global network. This built-in redundancy means that if an individual connection, server, or data center goes down, your origin remains available.
```

File: content/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms.md (hunk 1 above)
Comment (ranbel) [inline content/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms.md]:
```suggestion
You can create and configure a tunnel once and run that tunnel through multiple, unique instances of the connector, `cloudflared`. These instances are known as replicas. DNS records and Cloudflare Load Balancers will still point to the tunnel and its DNS Record (`UUID.cfargotunnel.com`), while that tunnel sends traffic to the multiple instances of `cloudflared` that run through it. Today, there is no guarantee about which replica will be chosen. Replicas are often deployed to provide tunnels with high availability in the event a given host running `cloudflared` is interrupted or taken offline. 
```

**Pull request merged**